TransferMethod/Sources/Extensions/HyperwalletTransferMethodUI.swift

```diff
@@ -85,4 +86,24 @@ public extension HyperwalletUI {
         coordinator.start(initializationData: initializationData, parentController: parentController)
         return coordinator
     }
+
+    /// Controller to update a transfer method.
+    ///
+    /// The form fields are based transfer method should be
+    /// passed to this Controller to update Transfer Method for those values.
+    ///
+    /// - Parameters:
+    ///   - transferMethod: The transfer method being edited.
+    ///   - forceUpdateData: Forces to refresh the cached data.
+    /// - Returns: An instance of `AddTransferMethodCoordinator`
+    func updateTransferMethodCoordinator(_ transferMethod: HyperwalletTransferMethod,
```

Comment:
this should be just transferMethodToken: String (token of TransferMethod to be updated)

Reply:
Updated